modules/sprint_reagent_aliquot_labels.js

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
+import config from '@/nuxt.config'
+import { query, headers } from '@/modules/sprint_constants'
+
+// Will create a new layout object for a print job
+// Requires barcode and two lines of text
+// TODO: how do we turn this into external json
```

Comment:
Is that TODO unresolved

Reply:
Yes indeed.  This is not external JSON.  I think the suggestion here is that it becomes a config item.  Copied the TODO from the existing sprint module.